frontend/packages/gitops-plugin/src/components/details/GitOpsDetails.tsx

```diff
@@ -179,7 +176,7 @@ const GitOpsDetails: React.FC<GitOpsDetailsProps> = ({ envs, appName }) => {
                                   <span className="odc-gitops-details__env-section__argo-external-link">
                                     <img
                                       loading="lazy"
-                                      src={ArgoCDIcon}
+                                      src="https://cd.apps.argoproj.io/assets/images/logo.png"
```

Comment:
As I mentioned in a DM to Regina; if this is not available in offline envs, we probably should look to get another icon.